apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -1308,10 +1308,10 @@ var (
 			CoreDNS:                   m("coredns/coredns:1.2.6"),
 			CoreDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.0.0"),
 		},
-		// Enabled in Rancher v2.2.9
-		"v1.13.12-rancher1-1": {
+		// Enabled in Rancher v2.2.13
+		"v1.13.12-rancher2-2": {
 			Etcd:                      m("quay.io/coreos/etcd:v3.2.24-rancher1"),
-			Kubernetes:                m("rancher/hyperkube:v1.13.12-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.13.12-rancher2"),
```

Comment:
Why is this `v1.13.12-rancher2`?

Reply:
it's based on the latest 1.13.x that exists currently https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.5/rke/k8s_rke_system_images.go#L1097